Sources/View/NibOwnerLoadable.swift

```diff
@@ -38,27 +38,25 @@ public extension NibOwnerLoadable where Self: UIView {
 
    - parameter owner: The instance of the view which will be your File's Owner
                       (and to which you want to add the XIB's views as subviews).
-                      Defaults to a brand new instance if not provided.
-   - returns: A `NibOwnLoadable`, `UIView` instance
+   - returns: A `NibOwnerLoadable`, `UIView` instance
    */
   @discardableResult
-  static func loadFromNib(owner: Self = Self()) -> Self {
```

Comment:
_huh at first I thought my comment on this line had disappeared, so I reposted it as you saw above… but now I see it's present twice… on the same line… but presented as 2 separated comments if it was 2 different files?! GitHub bug?!_